woocommerce/woocommerce-gateway-stripe
# Address QIT Security test errors

Fixes #3255

## Changes proposed in this Pull Request:

The security test errors reported by the QIT have been fixed in this PR. It also adds workflow files required to run the QIT test using labels.

![image](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/25176325/9e96a2a8-685e-4b26-9c4f-92d152415faf)
[link](https://qit.woo.com/?qit_results=4627217.xa1utvI8GAEj7acP40nusETvny2xmsI20pGWxteWvDLYffIOOiLWxke4VwcfEpFv)

Note: We'll need to update the branch name from the workflow file [here](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3260/files#diff-afc8dd44a564984aed3ebea479fa2615e9f5e586ffc067fe717d0aa8a6877484R33) before merging.

## Testing instructions

1. QIT Security Test GH action should be passed.
2. Regression testing of the extension.

---


**Post merge**


## Changelog

Fix - Address QIT Security test errors
